doxygen/input/doc_mainpage.h

```diff
@@ -5,7 +5,7 @@
  */
 /*
  *
- *  Copyright (C) 2006-2019, ARM Limited, All Rights Reserved
+ *  Copyright (C) 2006-2015, ARM Limited, All Rights Reserved
```

Comment:
This change will not be undone by the merge because 2015 is what's in mbedtls. We should update copyright years when we modify a file, but that's a preexisting issue which is out of scope here.